DEVELOPMENT.md

```diff
@@ -63,6 +63,28 @@ When working on a feature, the order you work on things matter. Think of a featu
 - Minor Annoyances: Things that hurt the user experience but the person can get over. Example would be an error message popping up in the middle but if they hit ok they can keep going
 - Minor Refactoring
 
+### Should I make my code more general?
+Sometimes we consider adding more flexibility to our current code in order to make future changes easier or to make a piece of code usable for some slightly different scenarios, this process has a couple issues:
+
+-  **The extra code you write may not be used at all:**
```

Comment:
These headings have ":" but the rest of the document doesn't. We need consistency.